### ISSUE TITLE
Need to set endPort

### ISSUE THREAD

**Issue by do-yeon** [Stocard/node-ssh-forward#12]
I tried to use it but there was problem that we didn't connect our server by using default port.
I think ssh2 can set custom port to connect server.
So, it's gonna work well with my request if you change a little thing.

**Comment by Junkern** [comment]
Let me rephrase: You want to be able to connect to a different server port than `22`? If so, yes, that should be easily doable :)

**Comment by do-yeon** [comment]
yes, exactly!!

**Comment by Junkern** [comment]
@do-yeon, published with https://github.com/Stocard/node-ssh-forward/releases/tag/v0.5.0

**Comment by richbayliss** [comment]
This doesn't seem to work for me in the latest version, and looking at the code it should be as easy as passing `this.options.endPort` to the `options` in `connection.connect(options);`

The use case here is having the SSH server be on a different port to 22, which right now I cannot get working.

**Comment by Junkern** [comment]
@richbayliss sorry for the long wait. I finally got around to work on that and just released https://github.com/Stocard/node-ssh-forward/releases/tag/v0.6.3
